antsibull/data/docsite/plugin.rst.j2

```diff
@@ -71,7 +74,16 @@
 {% if doc['deprecated'] -%}
 DEPRECATED
 ----------
+{% if 'removed_at_date' in doc['deprecated'] %}
+:Removed in: major release after @{ doc['deprecated']['removed_at_date'] | rst_ify }@
+{% elif 'removed_in' in doc['deprecated'] %}
 :Removed in: version: @{ doc['deprecated']['removed_in'] | rst_ify }@
+{% else %}
+:Removed in: a future release
+{% endif %}
+{% if (doc['deprecated']['removed_from_collection'] or collection_name) != collection_name %}
```

Comment:
I had to read this a few times... I think I'd prefer it written as:
```
{% if (doc['deprecated']['removed_from_collection'] and doc['deprecated']['removed_from_collection'] != collection_name %}
```

It's longer but using two clauses in the condition makes it clearer what we're checking for.  (Assuming that I did understand what we're checking for ;-)

Reply:
(Same for other places where this condition appears)

Reply:
Fixed.